google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ConnectionImpl.java

```diff
@@ -955,7 +957,12 @@ BigQueryResult highThroughPutRead(
 
     try {
       if (bqReadClient == null) { // if the read client isn't already initialized. Not thread safe.
-        bqReadClient = BigQueryReadClient.create();
+        BigQueryReadSettings settings =
+            BigQueryReadSettings.newBuilder()
+                .setCredentialsProvider(
+                    FixedCredentialsProvider.create(bigQueryOptions.getCredentials()))
```

Comment:
Is the FixedCredentialsProvider indirection needed, or can we set the retrieved credentials directly?  Is this a typing issue, or does the indirection get us some other benefit?